lambda/MAS/MAS.csproj

```diff
@@ -21,7 +21,7 @@
     <PackageReference Include="Amazon.Lambda.AspNetCoreServer" Version="3.1.0" />
     <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="2.1.10" />
     <PackageReference Include="Nager.Date" Version="1.25.9" />
-    <PackageReference Include="NICE.Logging" Version="6.0.9" />
+    <PackageReference Include="NICE.Logging" Version="6.0.22" />
```

Comment:
Can't comment on the binary...but it looks to me like the accreditation logo would be better off as a png (with the edge being transparent)